docker-compose.yml

```diff
@@ -108,15 +108,15 @@ services:
     environment:
       - LOCALSTACK_HOST=localstack
     ports:
-      - "4566:4566"
+      - "${EXPOSE_LOCALSTACK_PORT:-4566}:4566"
```

Comment:
How do you want to set the environment variables? If you set them in `.env`, they will only be used by the `Makefile`, and not if VS Code brings the container up. If you set them in your `.bashrc`, then all the variable names should contain `TECKEN`, so they don't conflict with variable names for other services.

Reply:
docker compose automatically picks up variables from .env https://docs.docker.com/compose/environment-variables/set-environment-variables/#substitute-with-an-env-file

I made sure it works by exiting vscode, running `docker compose -f docker-compose.yml -f .devcontainer/docker-compose.yml down`, opening vscode, waiting for the devcontainer to finish starting, and finally checking the exposed ports via `docker compose -f docker-compose.yml -f .devcontainer/docker-compose.yml ps`

Reply:
Oh, nice! I didn't know. We should make use of that for the build arguments. They are currently passed in explicitly from the `Makefile`, but we should add them to `docker-compose.yml` instead.